{"metadata":{"externalID":"s:9yasee_ios6YErrorO9unConnectyAcA6ErrObjVcACmF","role":"symbol","roleHeading":"Case","symbolKind":"case","title":"YError.unConnect(_:)","modules":[{"name":"yasee_ios"}],"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unConnect","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:9yasee_ios6ErrObjV","text":"ErrObj","kind":"typeIdentifier"},{"kind":"text","text":")"}]},"sections":[],"schemaVersion":{"patch":0,"major":0,"minor":3},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/yasee_ios\/yerror\/unconnect(_:)"]}],"hierarchy":{"paths":[["doc:\/\/com.yasee.yasee\/documentation\/yasee_ios","doc:\/\/com.yasee.yasee\/documentation\/yasee_ios\/YError"]]},"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"unConnect","kind":"identifier"},{"kind":"text","text":"("},{"kind":"externalParam","text":"_"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"ErrObj","preciseIdentifier":"s:9yasee_ios6ErrObjV","identifier":"doc:\/\/com.yasee.yasee\/documentation\/yasee_ios\/ErrObj"},{"kind":"text","text":" = .init(code: 0x04, message: \"设备未连接成功\", suggestion: \"请重新连接设备重试!\"))"}],"languages":["swift"]}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.yasee.yasee\/documentation\/yasee_ios\/YError\/unConnect(_:)"},"abstract":[{"type":"text","text":"设备未连接成功"}],"references":{"doc://com.yasee.yasee/documentation/yasee_ios":{"role":"collection","title":"yasee_ios","identifier":"doc:\/\/com.yasee.yasee\/documentation\/yasee_ios","url":"\/documentation\/yasee_ios","abstract":[],"kind":"symbol","type":"topic"},"doc://com.yasee.yasee/documentation/yasee_ios/YError":{"abstract":[{"text":"错误信息","type":"text"}],"role":"symbol","identifier":"doc:\/\/com.yasee.yasee\/documentation\/yasee_ios\/YError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"YError","kind":"identifier"}],"navigatorTitle":[{"text":"YError","kind":"identifier"}],"type":"topic","url":"\/documentation\/yasee_ios\/yerror","kind":"symbol","title":"YError"},"doc://com.yasee.yasee/documentation/yasee_ios/YError/unConnect(_:)":{"role":"symbol","kind":"symbol","abstract":[{"type":"text","text":"设备未连接成功"}],"url":"\/documentation\/yasee_ios\/yerror\/unconnect(_:)","identifier":"doc:\/\/com.yasee.yasee\/documentation\/yasee_ios\/YError\/unConnect(_:)","title":"YError.unConnect(_:)","type":"topic","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"unConnect","kind":"identifier"},{"kind":"text","text":"("},{"text":"ErrObj","kind":"typeIdentifier","preciseIdentifier":"s:9yasee_ios6ErrObjV"},{"kind":"text","text":")"}]},"doc://com.yasee.yasee/documentation/yasee_ios/ErrObj":{"type":"topic","url":"\/documentation\/yasee_ios\/errobj","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ErrObj"}],"identifier":"doc:\/\/com.yasee.yasee\/documentation\/yasee_ios\/ErrObj","kind":"symbol","navigatorTitle":[{"text":"ErrObj","kind":"identifier"}],"abstract":[{"type":"text","text":"错误内容"}],"title":"ErrObj","role":"symbol"}}}